src/operator/nn/mkldnn/mkldnn_act.cc

```diff
@@ -45,11 +45,9 @@ namespace op {
 bool SupportMKLDNNAct(const ActivationParam& param) {
   // We only enable ReLU for now. It seems other activations have some precision
```

Comment:
Remove comment

Reply:
Remove comment why?

Reply:
Because the comment explains why the operators have been disabled. This PR re-enables them and thus the comment is obsolete